### ISSUE TITLE
Missing dialog annotation for Modelica.Mechanics.Rotational.Sources.Speed.f_crit

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#821]
**Reported by anonymous on 14 Sep 2012 08:07 UTC**
`Modelica.Mechanics.Rotational.Sources.Speed.f_crit` misses the dialog annotation `Dialog(enable=not exact)` in order to comply with similar elements
```
Modelica.Mechanics.Rotational.Sources.Position
Modelica.Mechanics.Translational.Sources.Position
Modelica.Mechanics.Translational.Sources.Speed
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/821

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 14 Sep 2012 08:24 UTC**
FIXED in 3635625ae51c47e10a13d50876cd073f284fa3b2 for trunk (3.2.1 release)

**Comment by modelica-trac-importer** [comment]
**Changelog modified by dietmarw on 14 Sep 2012 08:24 UTC**
Missing `Dialog` annotation added .

**Comment by modelica-trac-importer** [comment]
**Comment by anonymous on 14 Sep 2012 08:40 UTC**
I thought that trunk is vs.3.3?? Please backport to /branches/maintenance/3.2 and branches/maintenance/3.2.1!

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 14 Sep 2012 08:42 UTC**
trunk is 3.2.1 for now and since this is not a critical bug but rather an enhancement there is no need to back port it to 3.2. One can rather use 3.2.1 once released.

**Comment by modelica-trac-importer** [comment]
**Comment by anonymous on 14 Sep 2012 08:50 UTC**
Does this mean that branches/maintenance/3.2 and branches/maintenance/3.2.1 are read-only, i.e. maintenance has stopped?

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 14 Sep 2012 14:32 UTC**
There will be a `branches/maintenance/3.2.1` once 3.2.1 is released. As for 3.2 it contains criticial bugfixes (check the log and you will see that) and is still maintained just not for uncritical bugfixes that will be available soon in 3.2.1 anyway.
